### DISCUSSION TITLE
0.1.6

### DISCUSSION THREAD

**Discussion by tim-hub** [tim-hub/obsidian-bible-reference#14] [Announcements]
## What's Changed
* UK English by @kmf in https://github.com/tim-hub/obsidian-bible-reference/pull/13

## New Contributors
* @kmf made their first contribution in https://github.com/tim-hub/obsidian-bible-reference/pull/13

**Full Changelog**: https://github.com/tim-hub/obsidian-bible-reference/compare/0.1.5...0.1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/tim-hub/obsidian-bible-reference/releases/tag/0.1.6'>0.1.6</a>.</em>

**Comment by kmf** [comment]
The update does **_not_** work via Obsidian

**Reply by tim-hub** [reply]
> The update does **_not_** work via Obsidian

Hi @kmf 

Would you mind sharing more details, because it does work on my side.

**Reply by tim-hub** [reply]
Like, obsidian version? language you chose?

**Comment by tim-hub** [comment]
Thanks @kmf for the feedback

It should work now. 

I made a a mistake when releasing, it broke the plugin, now it should be fixed now. Sorry for that.
